### ISSUE TITLE
Weighted metrics are not loaded using load_model, only normal metrics are set

### ISSUE THREAD

**Issue by wouterbulten** [keras-team/keras#10221]
When using sample weights its important to use `weighted_metrics` instead of `metrics` to get the correct accuracy. However, the current implementation of load/save_model does not take these metrics in to account. 

Current implementations:

- `save_model` only saves metrics, see https://github.com/keras-team/keras/blob/master/keras/engine/saving.py#L144
- `load_model` only sets metrics, see https://github.com/keras-team/keras/blob/master/keras/engine/saving.py#L286

This problem only occurs after a loading a model and then trying to continue training (or using one of the weighted metrics). Ideally, the weighted metrics should also be saved and subsequently loaded.

**Comment by Tejasvedagiri** [comment]
Even I faced the same issue . Is there any way to re load the metrics back to Network.

**Comment by Luux** [comment]
I can confirm this issue. When loading a saved model, weighted_metrics remains None.

**Comment by Luux** [comment]
I can confirm that this is still an issue. After loading an model, weighted_metrics remains None.
Steps to reproduce (example): 
1. build some model and compile with `model.compile(optimizer="Adam", loss='binary_crossentropy', metrics=['mae', 'acc'], weighted_metrics=['mae', 'acc'])`
2. train some stuff, evaluate -> weighted_metrics are given
3. save the model
4. load it
5. weighted_metrics are gone


-
